### PR TITLE
🐛  typographyTemplate in Tab

### LINKED ISSUE #2283: wrong/missing body/font-family reference for tabs
https://eds-storybook-react.azurewebsites.net/?path=/story/navigation-tabs--with-search

### PR DESCRIPTION
**Pull request by martalalik**
Resolves #2283

### CODE REVIEW

File: packages/eds-core-react/src/components/Tabs/Tab.tsx [hunk 1]
```diff
@@ -28,6 +29,7 @@ const StyledTab = styled.button.attrs<TabProps>(
     font-size: 1rem;
```
Comment (mimarz) [inline packages/eds-core-react/src/components/Tabs/Tab.tsx:29]: Suggest cleaning up the css thats now replaced by typography template. Font-family, font-size.

**Review by mimarz — CHANGES_REQUESTED** [review]
Looking good! Just some suggested cleaning up

File: packages/eds-core-react/src/components/Tabs/Tab.tsx (hunk 1 above)
Comment (martalalik) [reply packages/eds-core-react/src/components/Tabs/Tab.tsx:29]: Done :)

Comment (martalalik) [reply packages/eds-core-react/src/components/Tabs/Tab.tsx:29]: Done 😊

**Review by mimarz — APPROVED** [review]
LGTM 👍

**Pull request merged**